src/store/account.js

```diff
@@ -180,26 +180,32 @@ export default {
 			context.getters.timeline.setStore(context).initialFetch();
 		},
 
-		// Fetch data from the SDK By Address.
+		// Fetch data from the SDK By plain Address.
 		async fetchAccountDetail(context, payload) {
-			if (!helper.isAccountAddress(payload.address))
-				payload.address = await helper.decodeToAddress(payload.address);
```

Comment:
IMO the flow should be

- check if the value is plain address, then return it
- remove the hyphens and check if the value is plain address, then return it
- check if the value is publicKey, then convert to address and return it
- check (fetch) if the value is an address alias then return address

Reply:
The first check should be address or alias, right?